### PR TITLE
Implemented: mobile view support for the inprogress page of fulfillment app (#244)

### LINKED ISSUE #244: Add Mobile View Support to the App
## What is the motivation for adding/enhancing this feature?
Currently, the app only supports a desktop view, which limits its accessibility and user-friendliness on mobile devices. To enhance the user experience and reach a broader audience, we should implement a mobile view for the app.

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by amansinghbais**
### Related Issues

Related Issue: #244

### Short Description and Why It's Useful

Implemented the mobile view for the inprogress page in the fulfillment app.
- Implemented the EditPackagingModal and ReportIssueModal so as to display order details and items in the modal.
- Added the functionality to the modal's popovers and buttons.

### Screenshots of Visual Changes before/after (If There Are Any)

Before
Screenshot from 2023-08-31 18-19-58
Screenshot from 2023-08-31 18-26-35
Screenshot from 2023-08-31 18-30-51

After
Screenshot from 2023-08-31 18-23-38
Screenshot from 2023-08-31 18-26-20
Screenshot from 2023-08-31 18-26-04

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/ionic-bopis#contribution-guideline)

### CODE REVIEW

File: src/views/EditPackagingModal.vue [hunk 1]
```diff
@@ -124,17 +127,27 @@ export default defineComponent({
...
+  props: ['order', 'updateBox', 'addingBoxForOrderIds', 'addShipmentBox', 'save'],
   setup() {
     return {
       addCircleOutline,
       closeOutline,
-      pricetag
+      pricetag,
+      copyToClipboard,
+      formatUtcDate,
+      getFeature
```
Comment (k2maan) [inline src/views/EditPackagingModal.vue:150]: Follow alphabetical order for exports

File: src/views/ReportIssueModal.vue [hunk 2]
```diff
@@ -7,63 +7,61 @@
...
-        <ion-button fill="clear">{{ $t("Save") }}</ion-button>
+      <ion-buttons slot="end" @click="save(order)">
+        <ion-button color="primary" fill="clear">{{ $t("Save") }}</ion-button>
       </ion-buttons>
     </ion-toolbar>
   </ion-header>
 
  <ion-content>
-   <ion-card>
+    <ion-card>
```
Comment (k2maan) [inline src/views/ReportIssueModal.vue:17]: Fix indentation

File: src/views/EditPackagingModal.vue [hunk 3]
```diff
@@ -17,47 +17,45 @@
...
         </div>
       </div>
 
-      <div class="order-item">
+      <div v-for="(item, index) in order.items" :key="index" class="order-item">
         <div class="product-info">
           <ion-item lines="none">
             <ion-thumbnail slot="start">
-              <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
+              <Image :src="getProduct(item.productId).mainImageUrl" />
```
Comment (k2maan) [inline src/views/EditPackagingModal.vue:45]: We should use the ShopifyImage component with the image size as small.

File: src/views/PackagingPopover.vue [hunk 4]
```diff
@@ -41,22 +37,35 @@ export default defineComponent({
...
       return reportmodal.present();
     }
   },
+  props: ['order', 'updateBox', 'addingBoxForOrderIds', 'addShipmentBox', 'save', 'updateRejectReason'],
   setup() {
     return {
         pencil,
         warning,
-        refresh    
+        refresh
```
Comment (k2maan) [inline src/views/PackagingPopover.vue:68]: Fix indentation and follow alphabetical sorting

File: src/views/ReportIssueModal.vue [hunk 5]
```diff
@@ -7,63 +7,61 @@
...
-      <ion-item>
-        <ion-label>{{ $t("Select an issue") }}</ion-label>
-        <ion-select value="a">
-          <ion-select-option value="a">Out of stock</ion-select-option>
-          <ion-select-option value="b">Worn display</ion-select-option>
+      
+      <ion-item lines="none">
+        <ion-label>{{ $t("Select issue") }}</ion-label>
+        <ion-select interface="popover" @ionChange="updateRejectReason($event, item, order)" :value="item.rejectReason" >
+          <ion-select-option v-for="reason in rejectReasons" :key="reason.enumId" :value="reason.enumId">{{ reason.description ? $t(reason.description) : reason.enumId }}</ion-select-option>
```
Comment (k2maan) [inline src/views/ReportIssueModal.vue:64]: Translate` reason.description `only if all description entries are added in the label json files.

File: src/views/ReportIssueModal.vue [hunk 6]
```diff
@@ -90,34 +88,48 @@ import {
...
   },
+  props: ['order', 'save', 'updateRejectReason'],
   setup() {
     return {
       closeOutline,
-      pricetag
+      pricetag,
+      copyToClipboard,
+      formatUtcDate,
+      getFeature
```
Comment (k2maan) [inline src/views/ReportIssueModal.vue:132]: Follow alphabetical sorting

**Review by alsoK2maan — CHANGES_REQUESTED** [review]

**Review by alsoK2maan — APPROVED** [review]

**Comment by ymaheshwari1** [comment]
Closing as the apps UI is getting changed and there is no immediate requirement of mobile support.

**Pull request closed without merging**